README.md

```diff
@@ -53,242 +22,248 @@ Human Essentials is one of many projects initiated and run by Ruby for Good. You
+*Tip: If you need to skip a failing test, place `pending("Reason you are skipping the test")` into the `it` block rather than skipping with `xit`. This will allow rspec to deliver the error message without causing the test suite to fail.*
 
-Tip: If you need to skip a failing test, place `pending("Reason you are skipping the test")` into the `it` block rather than skipping with `xit`. This will allow rspec to deliver the error message without causing the test suite to fail.
-
-example:
 ```ruby
   it "works!" do
     pending("Need to implement this")
     expect(my_code).to be_valid
   end
 ```
 
-### On Writing Browser/System/Feature Specs
+### Writing Browser/System/Feature Specs
 
 If you need to see a browser/system spec run in the browser, you can use the following env variable:
 
 ```
 NOT_HEADLESS=true bundle exec rspec
 ```
 
-##### Use magic_test to simplify browser/system/feature spec writing
-We've added [magic_test](https://github.com/bullet-train-co/magic_test) which makes creating browser specs much easier. It does this by giving you the ability to record actions on the browser running the specs and easily paste them into the spec.
-
-For example you can do this by adding `magic_test` within your system spec:
+We've added [magic_test](https://github.com/bullet-train-co/magic_test) which makes creating browser specs much easier. It allows you to record actions on the browser running the specs and easily paste them into the spec. You can do this by adding `magic_test` within your system spec:
 ```rb
  it "does some browser stuff" do
    magic_test
  end
 ```
-and run the spec using this command:
-```
-MAGIC_TEST=1 NOT_HEADLESS=true bundle exec rspec <path_to_spec>
-```
+and run the spec using this command: `MAGIC_TEST=1 NOT_HEADLESS=true bundle exec rspec <path_to_spec>`
 
 **See videos of it in action [here](https://twitter.com/andrewculver/status/1366062684802846721)**
 
-# 🚀 Deployment Process
-The human-essentials & partner application should be deployed ideally on a weekly or bi-weekly schedule. However, this depends on the amount of updates that we have merged into main. Assuming there is updates that we want to ship into deploy, this is the process we take to getting updates from our `main` branch deployed to our servers.
+# Deployment Process 🚀
+The human-essentials & partner application should ideally be deployed on a weekly or bi-weekly schedule depending on the merged updates in the main branch. This is the process we take to deploy updates from our main branch to our servers.
+
+### Requirements
+- SSH access to our servers (usually granted to core maintainers)
+- Login credentials to our [Mailchimp](https://mailchimp.com/) account
+
+
+### Tag & Release
+1. Push a tag with the appropriate semantic versioning. Refer to the [releases](https://github.com/rubyforgood/human-essentials/releases) for the correct versioning. For example, if the last release was `2.1.0` and you're making a hotfix, use `2.1.1`
```

Comment:
We switched to date-based versioning, but I can update this section in a separate PR.

Reply:
Awesome, I'd like to leave that update for a separate PR. I opened a new [issue](https://github.com/rubyforgood/human-essentials/issues/3755#issue-1812212931) for this